launcher/main.js

```diff
@@ -116,6 +116,12 @@ if (!lock) {
 		enable_developer: false,
 		dev_modules_path: '',
 		log_level: 'info',
+
+		enable_syslog: false,
+		syslog_host: '127.0.0.1',
+		syslog_port: 514,
+		syslog_use_tcp: false,
+		syslog_local_hostname: os.hostname(),
```

Comment:
```suggestion
		syslog_local_hostname: '',
```
It defaults to using `os.hostname()` later on, so I dont think we should be baking the current hostname into the config file. It will just lead to it not following the current hostname unless the user explicitly clears that field